pom.xml

```diff
@@ -573,6 +486,89 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>com.coderplus.maven.plugins</groupId>
+                <artifactId>copy-rename-maven-plugin</artifactId>
+                <version>1.0.1</version>
+                <executions>
+                    <execution>
+                        <id>copy-properties</id>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
```

Comment:
Should I set here the `<phase>generate-sources</phase>` because it was set in the profiles before? Actually the plugin is run like expected.